RULES.md

```diff
@@ -0,0 +1,61 @@
+# Rule Documentation
```

Comment:
This is fine for configuration but we started a while a go this project to document each rule https://github.com/securego/securego.github.io. If you are interested in this area, you can contribute there. Thanks